isis/src/base/objs/ProcessByLine/ProcessByLine.cpp

```diff
@@ -70,19 +70,6 @@ namespace Isis {
   }
 
 
-  /**
-   * Set the InputCube vector to an opened Cube. This is used if there already
-   * exists a valid opened cube
-   *
-   * @author 2011-05-07 Sharmila Prasad
-   *
-   * @param inCube - Pointer to input Cube
-   */
-  void ProcessByLine::SetInputCube(Isis::Cube *inCube) {
-    Process::SetInputCube(inCube);
-  }
-
-
```

Comment:
Removed because it caused an error when changing `Pocess::SetInputCube` and is now redundant.